src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -5736,4 +5736,7 @@
   <data name="ERR_EmbeddedAttributeMustFollowPattern" xml:space="preserve">
     <value>The type 'Microsoft.CodeAnalysis.EmbeddedAttribute' must be non-generic, Friend, NotInheritable, have a Public parameterless constructor, inherit from System.Attribute, and be able to be applied to any type</value>
   </data>
+  <data name="ERR_MethodImplAttributeAsyncCannotBeUsed" xml:space="preserve">
+    <value>'MethodImplAttribute.Async' cannot be manually applied to methods.</value>
```

Comment:
Curious why the slightly different error message.  Why not mention adding the 'Async' modifier?